app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -24,26 +24,30 @@
 %>
 
 <%= render 'nav', step_by_step_page: @step_by_step_page, active: "internal-change-notes" %>
+
+<% displayed_edition_number = nil %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render "govuk_publishing_components/components/accordion", {
-      items: @step_by_step_page.internal_change_notes.map.with_index do |note, index|
-        {
-          heading: {
-            text: "#{note.readable_created_date}"
-          },
-          summary: {
-            text: sanitize("#{note.edition_number ? "Edition #{note.edition_number}" : "Current version"} - Change made by <strong>#{note.author}</strong>")
-          },
-          content: {
-            html: sanitize("
-              <pre class=\"govuk-body change-note--description\">#{note.description}</pre>
-            ")
-          },
-          expanded: index == 0
-        }
-      end
-    } %>
+    <% @step_by_step_page.internal_change_notes.each do |entry| %>
+      <div class="app-c-timeline-entry">
+        <% if displayed_edition_number != entry.edition_number %>
+          <h3 class="app-c-timeline-entry__title"><%= "#{entry.edition_number.to_i.ordinalize} edition" %></h3>
```

Comment:
What happens here if `entry.edition_number` is nil? Originally it displayed `Current version`.

Reply:
This is also not present in the final changes. It looks like this after the last commit `"#{records.first.edition_number ? "#{records.first.edition_number.to_i.ordinalize} edition" : "Current version"}"`

Reply:
To answer the original question, at this point in the code, the history looks like:

![Screen Shot 2019-08-21 at 11 43 10](https://user-images.githubusercontent.com/5793815/63426413-a6488b80-c40a-11e9-96a7-bbc3821d32fb.png)

Which is fine, but it's not clear that the first title is being lost for step by steps with a draft from the commit description.